.gitignore

```diff
@@ -41,8 +41,6 @@ wheels/
 *.egg-info/
 .installed.cfg
 *.egg
-.vscode/
-
```

Comment:
Not a fan of un-ignoring this file.
This will cause me to commit my local vscode settings.

Reply:
I kept the `.vscode` entry in line 130 so the directory will not be added to the repo.

Reply:
Yep - I see the `.vscode` on line 132. I verified that this ignores the folder contents still 
 
<img width="1744" alt="Screenshot 2023-06-12 at 11 49 34 AM" src="https://github.com/great-expectations/great_expectations/assets/2117313/d8b18cf6-8942-4b07-8586-ef1eb24bced6">